assets/wizards/site-design/views/theme-mods/index.js

```diff
@@ -0,0 +1,109 @@
+/**
+ * WordPress dependencies
+ */
+import { Component, Fragment } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import {
+	ImageUpload,
+	RadioControl,
+	TextControl,
+	ToggleControl,
+	ToggleGroup,
+	withWizardScreen,
+} from '../../../../components/src';
+import './style.scss';
+
+/**
+ * Theme Selection Screen.
+ */
+class ThemeMods extends Component {
+	/**
+	 * Render.
+	 */
+	render() {
+		const { themeMods, setThemeMods } = this.props;
+		const {
+			header_solid_background: headerSolidBackground,
+			header_simplified: headerSimplified,
+			header_center_logo: headerCenterLogo,
+			show_author_bio: authorBio,
```

Comment:
```suggestion
			show_author_bio: authorBio = true,
```
This is one of the two mods which have non-false defaults. Until it's set (here or in the customiser) it's value won't show up in the API response.

Reply:
Fixed in https://github.com/Automattic/newspack-plugin/pull/433/commits/00401ab41b9918d55a0e2d436ebcd0f505365b1c